doc/reference_guc.xml

```diff
@@ -151,18 +151,16 @@ SET postgis.gdal_datapath = 'C:/Program Files/PostgreSQL/9.3/gdal-data';</progra
                 <title>Examples</title>
                 <para>Set and reset <varname>postgis.gdal_enabled_drivers</varname></para>
 
-                <para>Sets backend for all new connections to database</para>
+                <para>Sets drivers for all new connections to database</para>
                 <programlisting>ALTER DATABASE mygisdb SET postgis.gdal_enabled_drivers TO 'GTiff PNG JPEG';</programlisting>
 
-                <para>Sets default enabled drivers for all new connections to server. Requires super user access and PostgreSQL 9.4+.
+                <para>Sets default enabled drivers for all new connections to whole database cluster. Requires super user access and PostgreSQL 9.4+.
                     Also note that database, session, and user settings override this.</para>
-                <programlisting>ALTER SYSTEM SET postgis.gdal_enabled_drivers TO 'GTiff PNG JPEG';
-SELECT pg_reload_conf();
-                </programlisting>
-
                 <programlisting>
-SET postgis.gdal_enabled_drivers TO 'GTiff PNG JPEG';
-SET postgis.gdal_enabled_drivers = default;
```

Comment:
These need to stay cause they represent change drivers just for the running session which is important.  e.g. you don't want to reconnect for changes to take effect and you are using GDAL raster functionality for intermediary - one time use to convert to postgis raster at which point you might not need these.

Reply:
Thank you, that makes sense. It was slightly confusing to me as it was below the `ALTER SYSTEM` but with no text / explanation in between. Do you want me to copy the format (session, db, system) used in `enable_outdb_rasters`?

Reply:
Yes that would be great.